libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -71,6 +71,7 @@ class AP_BattMonitor
         float       voltage;            // voltage in volts
         float       current_amps;       // current in amperes
         float       current_total_mah;  // total current draw since start-up
+        float       total_mwh;           // total energy draw since start-up
```

Comment:
Not to nitpick, but could this be named `consumed_mwh` or similar? Once the concept of a power sources max energy is also present this becomes a naming problem.